src/syntax-tree/module.ts

```diff
@@ -39,7 +39,7 @@ import { Reference, Scope } from "./scope";
 import { TypeChecker } from "./type-checker";
 import { VariableController } from "./variable-controller";
 
-const ERROR_HIGHLIGHT_COLOUR: [number, number, number, number] = [255, 153, 153, 0.5];
+const ERROR_HIGHLIGHT_COLOUR: [number, number, number, number] = [255, 153, 153, 0.8];
```

Comment:
please separate the code for error_highlighting, warnings, and the blocks. The settings (at least the colors) should be completely independent.